pkg/receive/handler.go

```diff
@@ -148,7 +149,7 @@ func NewHandler(logger log.Logger, o *Options) *Handler {
 		expBackoff: backoff.Backoff{
 			Factor: 2,
 			Min:    100 * time.Millisecond,
-			Max:    30 * time.Second,
+			Max:    o.MaxBackoff,
```

Comment:
Does backoff of 0 mean no backoff?

Reply:
Unfortunately not which is why I was leaning towards keeping this hidden while we experiment with it in prod and avoid the corresponding docs change or changing the underlying implementation.
 
Under the hood, its using https://github.com/jpillora/backoff so setting to 0 in this case actually sets it to 10s (which still worked way better than the previous 30s for us :) )